src/main/java/org/hl7/davinci/priorauth/Database.java

```diff
@@ -366,7 +404,8 @@ public boolean update(String resourceType, Map<String, Object> constraintParams,
         maps.add(data);
```

Comment:
Looks like the code is within a comment; probably better style to delete it if it's not really there.

Reply:
Which part of the code is in a comment?